src/features/cartography/infrastructure/presentation/pages/cartography/cartography.page.html

```diff
@@ -1,8 +1,24 @@
 <ng-container *ngIf="cnfsMarkers$ | async as cnfsMarkers">
-  <leaflet-map
-    [mapOptions]="mapOptions"
-    [cnfsMarkers]="cnfsMarkers"
-    [usagerCoordinates]="usagerCoordinates$ | async"
-    (addressToGeocode)="geocodeUsagerPosition($event)"
-    (usagerAutolocate)="updateUsagerPosition($event)"></leaflet-map>
+  <ng-container *ngIf="viewBox$ | async | viewCulling as visibleMarkers">
+    <div class="fr-grid-row">
+      <div class="fr-col-4">
+        <div class="fr-grid-row todo-alt-1" style="height: 15vw">
+          <address-geolocation
+            (addressToGeocode)="geocodeUsagerPosition($event)"
+            (usagerAutolocate)="updateUsagerPosition($event)"></address-geolocation>
+        </div>
+        <div class="fr-grid-row">Nombre de marqueurs présents visibles : {{ visibleMarkers.features.length }}</div>
+      </div>
+      <!--todo: fix calc-->
+      <div class="fr-col-8" style="height: calc(100vh - 173px)">
+        <leaflet-map
+          [mapOptions]="mapOptions"
+          [markers]="visibleMarkers | addUsagerMarker:(usagerCoordinates$ | async)"
+          stateChange
+          (viewreset)="mapViewChanged($event)"></leaflet-map>
```

Comment:
```suggestion
          (stateChange)="mapViewChanged($event)"></leaflet-map>
```
À faire suite à la modification dan sla directive stateChange